mw-webapp/src/helpers/getDataCy.ts

```diff
@@ -0,0 +1,4 @@
+/**
+ * Return [data-cy=value] for CYPRESS testing
```

Comment:
I think you can replace [...] as data attribute

Reply:
Done.